frontend/src/pages/destructionlist/hooks.ts

```diff
@@ -78,14 +82,17 @@ export function useDataGridProps(
     );
   }, []);
 
-  const fields = getFields(searchParams, zaaktypeChoicesState).map((field) => {
-    const isActiveFromStorage = fieldSelectionState?.[field.name];
-    const isActive =
-      typeof isActiveFromStorage === "undefined"
-        ? field.active !== false
-        : isActiveFromStorage;
-    return { ...field, active: isActive } as TypedField;
-  });
+  const hasAction = paginatedResults.results.some((zaak) => zaak.action);
```

Comment:
It feels to me that this implements specific logic (specific for the review page) in a generic piece of code (the hook building the base props for all datagrids).

We should not stuff in specific logic in generic logic this way. Instead of adding a bunch of if we could extend this prop in review page, adding our column there.